installers/install.sh

```diff
@@ -382,6 +382,11 @@ update_rc_file() {
   echo "$pathenv" >> "$RC_FILE"
 }
 
+# Write install file
+STATEPATH=$INSTALLDIR/$STATEEXE
+CONFIGDIR=$($STATEPATH "export" "config" "--filter=dir")
+echo "install.sh" >> $CONFIGDIR/"install.txt"
```

Comment:
I think that we should probably replace the file here, as otherwise (if people install the state tool several times without cleaning the config directory) we could see installation sources "install.sh\ninstall.sh\install.sh".

```suggestion
echo "install.sh" > $CONFIGDIR/"install.txt"
```